### DISCUSSION TITLE
Use with Friend/s

### DISCUSSION THREAD

**Discussion by SearchingSomeStuff** [jagrosh/MusicBot#1615] [Questions] [answered]
Guys, can you make 2 id's? Like you and your friend so your friend can also use it?
in the config file. If yes, then how?

**Comment by jagrosh** [comment, marked as answer]
The owner ID is only needed for the owner/host-level commands like changing the bot's name, avatar, etc. These should only ever be accessible to the person hosting the bot and owning the bot account.

Any other command levels (Admin, DJ) can be given to anyone via setting roles.

**Reply by SearchingSomeStuff** [reply]
Yeah, i already found it out. Still thanks! :D
